posts/2020-10-23-release-post-200011.adoc

```diff
@@ -0,0 +1,153 @@
+    libertyRuntime group: 'io.openliberty', name: 'openliberty-runtime', version: '[20.0.0.11,)'
+}
+----
+
+Or if you're using Docker:
+
+[source]
+----
+FROM open-liberty
+----
+//end::run[]
+
+Or take a look at our link:{url-prefix}/downloads/[Downloads page].
+
+[link=https://stackoverflow.com/tags/open-liberty]
+image::img/blog/blog_btn_stack.svg[Ask a question on Stack Overflow, align="center"]
+
+//tag::features[]
+
+[#kerberos]
+=== Kerberos authentication for JDBC data sources
+
+Kerberos is a network authentication protocol in which a client and server authenticate by communicating with a third process called a Key Distribution Center (KDC). Kerberos authentication may be used for JDBC data sources that are backed by one of the following databases:
+
+* IBM DB2
+* Oracle Database
+* Microsoft SQLServer
+
+Kerberos authentication in Liberty builds on top of the link:https://docs.oracle.com/en/java/javase/11/docs/api/jdk.security.auth/com/sun/security/auth/module/Krb5LoginModule.html[Kerberos Login Module and JGSS API provided by the JDK], which in turn builds on top of the Kerberos OS libraries for the specific system being used.
+
+The `kerberos` configuration element in `server.xml` provides system-wide configuration options for the Liberty server. For example:
+
+[source, xml]
+----
+  <kerberos keytab="${server.config.dir}/security/krb5.keytab" configFile="${server.config.dir}/security/krb5.conf"/>
+----
+
+Kerberos authentication is supported by a number of public features, including the following:
+
+* acmeCA-2.0
+* appSecurity-2.0/3.0/4.0
+* connectors-2.0
+* jdbc-4.0/4.1/4.2/4.3
+* jwt-1.0
+* mpMetrics-1.0/1.1/2.0/2.2/2.3/3.0
+* oauth-2.0
+* openapi-3.1
+* restConnector-2.0
+* socialLogin-1.0
+
+Below is an example of the Kerberos protocol being used to secure a `jdbc-4.2` data source: 
+
+[source, xml]
+----
+  <featureManager>
+    <feature>jdbc-4.2</feature>
+  </featureManager>
+
+  <!-- optional config: This is only needed if you need to customize the location of keytab or krb5.conf -->
+  <kerberos keytab="${server.config.dir}/security/krb5.keytab" configFile="${server.config.dir}/security/krb5.conf"/>
+
+  <authData id="myKerberosAuth" krb5Principal="krbUser"/>
+
+  <library id="db2DriverLib">
+    <fileset dir="${server.config.dir}/db2"/>
+  </library>
+
+  <dataSource jndiName="jdbc/krb/basic" containerAuthDataRef="myKerberosAuth">
+    <jdbcDriver libraryRef="db2DriverLib"/>
+    <properties.db2.jcc databaseName="${DB2_DBNAME}" serverName="${DB2_HOSTNAME}" portNumber="${DB2_PORT}"/>
+  </dataSource>
+----
+
+Before this release, it was technically possible to use Kerberos authentication with JDBC data sources, but the configuration was complex and un-documented. There was a lack of connection pooling support when using Kerberos authentication for data sources.
+
+Full documentation for this capability will be available soon in the link:{url-prefix}/docs/latest/overview.html[Open Liberty documentation] pages.
```

Comment:
@lauracowen do we want to make a reference to non-existing docs?

Reply:
Not really. I'm happy to comment it out for now and we can add it back in when we publish those docs (targetted at 20.0.0.12).

Reply:
Line has been commented out.